app/helpers/template_helpers.py

```diff
@@ -260,7 +287,7 @@ def safe_content(content):
     return content
 
 
-def get_data_layer(schema_theme):
+def get_data_layer(schema_theme: str) -> list:
```

Comment:
```suggestion
def get_data_layer(schema_theme: str) -> List:
```

It is preferable to use `typing.List` over the `list`: https://docs.python.org/3/library/typing.html#typing.List